src/features/game/types/withdrawables.ts

```diff
@@ -125,7 +136,25 @@ export const WITHDRAWABLES: Record<InventoryItemName, WithdrawCondition> = {
   "Mysterious Head": true,
   "Golden Bonsai": true,
   "Wicker Man": true,
+  "Engine Core": true,
```

Comment:
This was a quest item that is now legacy. We can keep this as `disabled` as there isn't a need to trade it